src/components/order-info/order-info.jsx

```diff
@@ -1,34 +1,28 @@
-export function OrderInfo({ title, orderInfo }) {
-  const tableContainer = {
-    display: "flex",
-    justifyContent: "space-between",
-    marginBottom: "12px",
-    fontSize: "16px",
-    fontWeight: "500",
-  };
+import styles from "./order-info.module.css";
 
+export function OrderInfo({ title, orderInfo }) {
   return (
     <>
-      <h1 style={{ fontSize: "20px", marginBottom: "8px" }}>{title}</h1>
-      <hr style={{ marginBottom: "35px" }} />
+      <h1 className={styles.title}>{title}</h1>
+      <hr className={styles.seperator} />
 
       {orderInfo.map(([detail, amount], index) => (
         <div
           key={index}
-          style={{
-            ...tableContainer,
-            ...(index === orderInfo.length - 1 ? { fontWeight: "bold" } : {}),
-          }}
+          className={`${styles.tableContainer} ${
+            index === orderInfo.length - 1 ? styles.bold : ""
```

Comment:
this operation shouldn't be online should be saved to the top of the component then use that variable here